### PR TITLE
Bump psammead styles

### PR DESCRIPTION
**Pull request by rhenshaw56**
part of #3814 

**Overall change:** _Bumps psammead styles._

**Code changes:**

- _Bump psammead styles in dependent packages._

---

- [X] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-radio-schedule/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 2.0.0-alpha.0 | [PR#3843](https://github.com/bbc/psammead/pull/3843) Bump Dependencies - @bbc/psammead-styles alpha version |
```
Comment (karinathomasbbc) [inline packages/components/psammead-radio-schedule/CHANGELOG.md]:
```suggestion
| 4.0.0-alpha.0 | [PR#3843](https://github.com/bbc/psammead/pull/3843) Bump Dependencies - @bbc/psammead-styles alpha version |
```

File: packages/components/psammead-radio-schedule/package-lock.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-radio-schedule",
-  "version": "3.0.26",
+  "version": "2.0.0-alpha.0",
```
Comment (karinathomasbbc) [inline packages/components/psammead-radio-schedule/package-lock.json]:
```suggestion
  "version": "4.0.0-alpha.0",
```

File: packages/components/psammead-radio-schedule/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-radio-schedule",
-  "version": "3.0.26",
+  "version": "2.0.0-alpha.0",
```
Comment (karinathomasbbc) [inline packages/components/psammead-radio-schedule/package.json]:
```suggestion
  "version": "4.0.0-alpha.0",
```

**Review by karinathomasbbc — APPROVED** [review]

**Review by chewu — APPROVED** [review]

File: package-lock.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead",
-  "version": "3.0.4",
+  "version": "4.0.0-alpha.0",
```
Comment (simonsinclair) [inline package-lock.json:3]: Alpha-version packages should probably have the following in their package.json:
```
"publishConfig": {
  "tag": "alpha"
}
```
This means they will be published with an `alpha` distribution tag. I.E. `npm publish --tag alpha`.

Comment (rhenshaw56) [reply package-lock.json:3]: Thanks @simonsinclair. I'll update these now

Comment (rhenshaw56) [reply package-lock.json:3]: @simonsinclair done.

Comment (simonsinclair) [reply package-lock.json:3]: Nice one, Rowland!

**Pull request merged**